mmv1/third_party/terraform/resources/resource_dataproc_job.go.erb

```diff
@@ -8,7 +9,11 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+<% if version == "ga" -%>
 	"google.golang.org/api/dataproc/v1"
```

Comment:
If we remove it altogether is `goimports` able to figure it out? I'm assuming it's not?

Reply:
Nope, it doesn't seem like it's able to figure it out